asklib.module

```diff
@@ -124,6 +124,47 @@ function asklib_cron() {
       $question->save();
     }
   }
+
+  // Clear contact information of questions older than 2 years every day after 1 am.
```

Comment:
I don't think this comment needs to have the specific periods since that information can be seen easily enough in the code and a comment like this can often become outdated.

Maybe just have this or something like it:
```suggestion
  // Clear contact information of old questions periodically to improve privacy.
```